.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to CDN
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version, without v prefix'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.version }}
+      - name: Use Node.js 14.x
+        env:
```

Comment:
Why are you injecting secrets into the environment of this step? First, it's a third-party action so that bears risk. But, also, I'm assuming you need the secrets in the final step that runs the `node` script? 🤔

Reply:
You're right, I'm not too familiar with github actions so I wasn't sure how this actually worked, should be right now.

Reply:
Thanks 😁  https://github.com/ably/ably-js/pull/889/commits/04ff346c729c68e371eeada6a4f41aeaa13c8882